lib/services/elasticsearch.js

```diff
@@ -982,7 +986,8 @@ function getElasticsearchRequest(request, kuzzle) {
   }
 
   ['from', 'size', 'scroll', 'scrollId', 'refresh'].forEach(argumentName => {
-    if (typeof request.input.args[argumentName] !== 'undefined') {
+    if (typeof request.input.args[argumentName] !== 'undefined' &&
+        extraParams.includes(argumentName)) {
```

Comment:
You can still apply @ballinette's solution without breaking unit tests, simply test if `extraParams` is defined prior to iterate over it.